packs/SoD.json

```diff
@@ -128,6 +253,68 @@
             "text": "No attachments except <i>Weapon</i>.\nIncrease the initiative value on your revealed plot card by 1 for each plot card in your used pile.",
             "deckLimit": 3
         },
+        {
+            "code": "10013",
+            "type": "character",
+            "name": "Loreza Sand",
+            "unique": true,
+            "faction": "martell",
+            "loyal": true,
+            "cost": 2,
+            "icons": {
+                "military": false,
+                "intrigue": true,
+                "power": false
+            },
+            "strength": 2,
+            "traits": [
+                "Bastard",
+                "Sand Snake"
+            ],
+            "text": "<b>Interrupt:</b> When a unique [[martell]] character you control is killed, draw 1 card. (Limit once per phase.)",
```

Comment:
Icons within text should be in single brackets - `[martell]` instead of `[[martell]]`. I'll add some validation rules around that for future PRs.